java/src/jni/hdfsdsImp.c

```diff
@@ -1441,6 +1442,7 @@ Java_hdf_hdflib_HDFLibrary_SDgetcompress(JNIEnv *env, jclass clss, jlong sdsid,
 done:
     return JNI_TRUE;
 }
+#endif /* H4_NO_DEPRECATED_SYMBOLS */
```

Comment:
@byrnHDF - Should I exclude it like this or move the ifdef inside the function and have it raise an exception?